src/main/java/software/amazon/smithy/lsp/ext/SmithyBuildExtensions.java

```diff
@@ -0,0 +1,145 @@
+package software.amazon.smithy.lsp.ext;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.Reader;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.stream.JsonWriter;
+
+/**
+ * 
+ * { ... normal smithy-build.json stuff ... "extensions": { "repositories":
+ * ["maven"] } }
+ */
+
+public final class SmithyBuildExtensions {
```

Comment:
Let's follow AWS best practice and replace this with an immutable POJO accompanied by a builder. just copy [this file](https://github.com/awslabs/smithy/pull/943/files#diff-1ac58bd502c28113b0adcdfd6cda1f422db25326345aa541393548247ac9babe) and remove anything irrelevant